src/savegame.cpp

```diff
@@ -440,6 +441,41 @@ void overmap::convert_terrain( const std::unordered_map<tripoint, std::string> &
                 nearby.push_back( { -1, school + "6", -1, school + "8", school_1 + "9_west" } );
             }
 
+        } else if( old == "prison_b_entrance" ) {
+            if (pos.z < 0) {
+                if (is_ot_type("prison_2", get_ter(pos.x, pos.y, pos.z + 1))) {
+                    new_id = oter_id("prison_b_2");
+                }
+            }
+
+        } else if( old == "prison_b" ) {
+            if (pos.z < 0) {
+                if (is_ot_type("prison_1", get_ter(pos.x, pos.y, pos.z + 1))) {
+                    new_id = oter_id("prison_b_1");
+                }
+                else if (is_ot_type("prison_3", get_ter(pos.x, pos.y, pos.z + 1))) {
+                    new_id = oter_id("prison_b_3");
+                }
+                else if (is_ot_type("prison_4", get_ter(pos.x, pos.y, pos.z + 1))) {
+                    new_id = oter_id("prison_b_4");
+                }
+                else if (is_ot_type("prison_5", get_ter(pos.x, pos.y, pos.z + 1))) {
+                    new_id = oter_id("prison_b_5");
+                }
+                else if (is_ot_type("prison_6", get_ter(pos.x, pos.y, pos.z + 1))) {
+                    new_id = oter_id("prison_b_6");
+                }
+                else if (is_ot_type("prison_7", get_ter(pos.x, pos.y, pos.z + 1))) {
+                    new_id = oter_id("prison_b_7");
+                }
+                else if (is_ot_type("prison_8", get_ter(pos.x, pos.y, pos.z + 1))) {
+                    new_id = oter_id("prison_b_8");
+                }
+                else if (is_ot_type("prison_9", get_ter(pos.x, pos.y, pos.z + 1))) {
```

Comment:
Styling:
```
} else if( x ) {
```
should be on one line.
`if` parentheses should be padded on the inside, but get no space between `if` and opening parenthesis.